app/assets/javascripts/sitewide/iframe_modal.js

```diff
@@ -24,4 +24,12 @@ $(document).on('turbolinks:load', function() {
     iFrameModalFrame.removeAttribute('src');
     iFrameModal.dispatchEvent(new Event('hidden'));
   });
+
+  iFrameModal.addEventListener('shown', () => {
+    document.body.style.overflow = 'hidden';
+  });
+
+  iFrameModal.addEventListener('hidden', () => {
+    document.body.style.overflow = 'auto';
+  });
```

Comment:
maybe it would be better to add a class to the body in this case, but this is also ok